### PR TITLE
Fix indentation on line 28 of `action.yml`

### LINKED ISSUE #43: YAML token indentation error in action.yml
### Description

There is an error in the `action.yml` file on line 28. The parameter is with an extra space.

### Screenshots

image

### Additional information

This bug was introduced by myself in #

### PR DESCRIPTION
**Pull request by guibranco**
## Fixes Issue

Closes #43 

## Check List (Check all the applicable boxes)

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Screenshots

Error:
image

Success:
image

## Note to reviewers

This bug was introduced by myself in #40

### CODE REVIEW

**Comment by guibranco** [comment]
> Stale pull request message

This is not stale, waiting for approval!

**Comment by guibranco** [comment]
@eddiejaoude this was waiting for review...

**Review by eddiejaoude — APPROVED** [review]
Sorry missed the notifications. Thank you 👍

**Pull request merged**
